### DISCUSSION TITLE
邀请 contributor 加入 Org

### DISCUSSION THREAD

**Discussion by LinuxSuRen** [opensource-f2f/episode#266] [Show and tell]
欢迎大家邀请**开源面对面**嘉宾以及以任何形式参与并有一定贡献的小伙伴加入我们的 GitHub Org，并给出推荐理由。

**Comment by LinuxSuRen** [comment]
考虑到 @xiaowangxiaowang256256 近期通过 issue 和 PR 的方式帮助改善官网，特推荐其加入。下面是相关链接：

* [4 个 issue](https://github.com/opensource-f2f/website/issues/created_by/xiaowangxiaowang256256)
* [5 个 PR](https://github.com/opensource-f2f/website/pulls?q=is%3Apr+author%3Axiaowangxiaowang256256)

![image](https://github.com/opensource-f2f/episode/assets/1450685/f1a3da39-af42-4394-ba44-bab27abfc30e)
![image](https://github.com/opensource-f2f/episode/assets/1450685/17446d2b-603e-4798-bc65-5d48e6906077)

如果您也认同 @xiaowangxiaowang256256 的 contribution，请帮忙给个 👍（攒够 3 个后发出邀请）。

**Reply by wey-gu** [reply]
欢迎 @xiaowangxiaowang256256 的加入，感谢你的热切贡献！

cc @sxzz

**Comment by xiaowangxiaowang256256** [comment]
我很开心能够获得 Rick 的 GitHub Org 推荐提名，也希望未来能和社区伙伴们一起，愉快地遨游在这开源世界中。

**Reply by LinuxSuRen** [reply]
![image](https://github.com/opensource-f2f/episode/assets/1450685/be0fd88a-73a7-43bc-9332-28385853c8b1)

**Reply by xiaowangxiaowang256256** [reply]
I am glad to join opensource-f2f Org.

**Comment by MikeyWei** [comment]
Welcome !
